Source/effects.h

```diff
@@ -12,6 +12,112 @@
 
 namespace devilution {
 
+enum class HeroSpeech {
+	_1,

```

Comment:
Well this made me look up which identifiers are reserved in C++: https://en.cppreference.com/w/cpp/language/identifiers

Let's replace the `_` with `N` or `I` to avoid raising eyebrows?